packages/mobile/e2e/src/usecases/RestoreAccountOnboarding.js

```diff
@@ -28,17 +26,9 @@ export default RestoreAccountOnboarding = () => {
     await element(by.id('AcceptTermsButton')).tap()
   })
 
-  it('Name and Number', async () => {
+  it('Name and Picture', async () => {
     await element(by.id('NameEntry')).replaceText(EXAMPLE_NAME)
-
-    await element(by.id('CountrySelectionButton')).tap()
-    await element(by.id('SearchInput')).replaceText(VERIFICATION_COUNTRY)
-    await element(by.id('Country_DE')).tap()
-
-    await expect(element(by.id('PhoneNumberField'))).toBeVisible()
-    await element(by.id('PhoneNumberField')).replaceText(VERIFICATION_PHONE_NUMBER)
-
-    await element(by.id('NameAndNumberContinueButton')).tap()
```

Comment:
This is part of verification now, that we don't have e2e tests for because we can't get over the captcha. Once that's resolved we'll need to add this again.

I tried to do the setting of the picture, but we can't handle the part where a user chooses the image from the gallery because that's outside of the app. I'll try to see if we can maybe mock that part